system/hubcopter/templates/_utils.tpl

```diff
@@ -0,0 +1,55 @@
+{{- define "glas_image" -}}
+  {{ $.Values.global.registry }}/hubcopter:{{ $.Values.glas.image_tag | required ".Values.hubcopter.image_tag is missing" }}
+{{- end -}}
+{{- define "hubcopter_image" -}}
+  {{ $.Values.global.registry }}/hubcopter:{{ $.Values.hubcopter.image_tag | required ".Values.hubcopter.image_tag is missing" }}
+{{- end -}}
+
+{{- define "glas_environment" }}
```

Comment:
Since you're only using these sets of environment variables once, you can inline this template into the respective Deployment spec. I'm usually only templating this when a set of environment variables is shared by multiple Deployments (e.g. tenso-api with tenso-worker).